src/main/webapp/location/geolocation.stories.js

```diff
@@ -84,3 +90,103 @@ Object.keys(editors).forEach(key => {
     )
   })
 })
+
+const keywordSuggestions = [
+  {
+    id: '1dc06d71-dc20-44d5-b211-de33816071c1',
+    name: 'Italy',
+  },
+  {
+    id: 'c0ab17cc-cc60-4a15-9860-3915aa9f0afb',
+    name: 'Italy Valley Cemetery',
+  },
+  {
+    id: 'cff9dea4-2e5e-4c03-9f5b-45495c638e78',
+    name: 'Italy Hill Cemetery',
+  },
+  {
+    id: '19e138c6-03df-47ea-a657-17b5cd2932df',
+    name: 'Italy Post Office',
+  },
+  {
+    id: '9573742f-dd27-4f25-ac83-25250269f469',
+    name: 'Italy Police Department',
+  },
+  {
+    id: 'cdbcd443-c987-489e-9e01-da21e82bc9a3',
+    name: 'Italy Naples Church',
+  },
+  {
+    id: '0816671b-4ce5-45c9-a045-8f83451bcbd5',
+    name: 'Italy Division',
+  },
+  {
+    id: 'd0ef5f36-681a-4cb6-90cc-1f344e594100',
+    name: 'Italy Mine',
+  },
+  {
+    id: '07f74c89-a7b2-4bf9-87fa-22d7d21a6ef4',
+    name: 'Italy Mine',
+  },
+  {
+    id: '75b3f19c-0872-466c-ad89-c051e832362e',
+    name: 'Italy Hollow Cemetery',
+  },
+]
+const keywordResult = {
+  type: 'Feature',
+  geometry: {
+    type: 'Polygon',
+    coordinates: [
+      [
+        [-77.3811, 42.5304],
+        [-77.1811, 42.5304],
+        [-77.1811, 42.7304],
+        [-77.3811, 42.7304],
+        [-77.3811, 42.5304],
+      ],
+    ],
+  },
+  properties: {},
+  id: 'Italy Valley Cemetery',
+}
+const extendedKeywordProps = {
+  getSuggestions: () => JSON.parse(JSON.stringify(keywordSuggestions)),
```

Comment:
Why are we `JSON.stringify` then `JSON.parse` here?

Reply:
To get a deep clone that isn't going to modify the original value and avoid any chances of side effects in the Storybook.

Reply:
Why is the suggestion at risk of being modified though?